test/tests/Pathfinding.cpp

```diff
@@ -97,7 +98,7 @@ class PathfindingTestBase : public testing::Test
         // destination will be picked, to try and get the peep towards the overall pathfinding goal.
         peep->PeepDirection = moveDir;
         auto destination = CoordsDirectionDelta[moveDir] + peep->GetLocation();
-        peep->SetDestination(destination, 2);
+        peep->SetDestination(destination);
```

Comment:
Any reason for removing the tolerance here?

Reply:
Looking at it, I think I might have removed it during a moment of misunderstanding, I'll add it back in my next commit, along with fixing the other things you've pointed out.